### PR TITLE
bifrost integration

### PR DESCRIPTION
**Pull request by akshay-ghy**
Replace crypto compare API calls to Biforst calls and only use it for daily ETLs.

### CODE REVIEW

File: ethereumetl/bifrost.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        access_token: str,
+
+) -> requests.Response:
+    base_url = f"https://dev.bifrost.palantree.com/coin-price-service/historical-daily-price/"
+    retries = Retry(backoff_factor=5, total=5)
+    adapter = HTTPAdapter(max_retries=retries)
+    session = requests.Session()
+    session.mount('https://', adapter)
+    session.mount('http://', adapter)
+    retries.BACKOFF_MAX = 600
```
Comment (saurabhdaga-merkle) [inline ethereumetl/bifrost.py:49]: I forgot, what is the inital sleep ?
2 secs?

Comment (akshay-ghy) [reply ethereumetl/bifrost.py:49]: 5 sec

**Comment by akshay-ghy** [comment]
Anything left here?

File: ethereumetl/bifrost.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+    return session.get(base_url, params=params)
+
+
+def get_coin_price(
+        from_currency_code: str,
+        execution_date: datetime.date,
+        to_currency_code: str = "USD",
+        access_token: str = BIFRSOT_AUTH,
+):
```
Comment (aniket818) [inline ethereumetl/bifrost.py:64]: spelling BIFRSOT_AUTH

File: ethereumetl/bifrost.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    retries = Retry(backoff_factor=5, total=5)
+    adapter = HTTPAdapter(max_retries=retries)
+    session = requests.Session()
+    session.mount('https://', adapter)
+    session.mount('http://', adapter)
+    retries.BACKOFF_MAX = 600
+    params = {
+        "fsyms": from_currency_code,
+        "tsyms": to_currency_code,
+        "access_token": access_token,
```
Comment (aniket818) [inline ethereumetl/bifrost.py:53]: fsyms to from_currencies
tsyms to to_currencies

**Pull request closed without merging**